### ISSUE TITLE
README is not very clear

### ISSUE THREAD

**Issue by g4s8** [artipie/npm-adapter#74]
The README file contains a lot of technical information about npm registry internals, but there are no contributors' entry point. We need to add these details:
 - What is npm-adapter (at the top of the readme file).
 - Usage: how to use `Npm` to update registry metadata (without serving a server), and how to integrate it to Artipie server.
 - Contribution: what are the main entry points of npm-adapter (where to start) and how they are related to each other, the short overall design description of this module. What are the primary integration tests.

**Comment by 0crat** [comment]
The job [#74](https://github.com/artipie/npm-adapter/issues/74) assigned to @brastak[/z](https://www.0crat.com/u/brastak), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/466c5789-531a-43bf-9da7-464abf04c86e); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); @brastak[/z](https://www.0crat.com/u/brastak) is not a member of this project yet, but they can request to join, as [§1](http://www.zerocracy.com/policy.html#1) explains; there will be no monetary reward for this job

**Comment by 0crat** [comment]
The user @brastak[/z](https://www.0crat.com/u/brastak) resigned from [#74](https://github.com/artipie/npm-adapter/issues/74), please stop working. Reason for job resignation: It is older than 10 days, see [§8](http://www.zerocracy.com/policy.html#8)

**Comment by 0crat** [comment]
The job [#74](https://github.com/artipie/npm-adapter/issues/74) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/a73b2678-ec55-4ce0-b727-88e708adaae3); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by victornoel** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/npm-adapter/issues/74#issuecomment-640464769))

@victornoel Job refused in 0 hours - no penalty, see [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
The job [#74](https://github.com/artipie/npm-adapter/issues/74) assigned to @marceloamadeu[/z](https://www.0crat.com/u/marceloamadeu), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/10388a1f-e0c8-4509-8c42-10e15b3a53d7); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by marceloamadeu** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/npm-adapter/issues/74#issuecomment-640747839))

@marceloamadeu The user @marceloamadeu[/z](https://www.0crat.com/u/marceloamadeu) resigned from [#74](https://github.com/artipie/npm-adapter/issues/74), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/npm-adapter/issues/74#issuecomment-640747839))

@marceloamadeu Job refused in 0 hours - no penalty, see [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by paulodamaso** [comment]
@g4s8 I think that this issue is outdated, seems that the information you require is already at readme.

**Comment by paulodamaso** [comment]
@0crat wait for @g4s8

**Comment by paulodamaso** [comment]
@g4s8 ping

**Comment by g4s8** [comment]
@paulodamaso I don't think so. It's still describing how `npm` CLI command works and what's the protocol of npm registry API, but there are no words about npm-adapter itself. Try to picture that you've just received a development task from this repository, what information you'd expect in the readme to quickly understand the project? Another case is if you receive a task from another module to integrate npm-adapter, what details about public API are missed here?

**Comment by g4s8** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/npm-adapter/issues/74#issuecomment-662291307))

@g4s8 The user @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) resigned from [#74](https://github.com/artipie/npm-adapter/issues/74), please stop working. Reason for job resignation: Order was cancelled
